src/corsheaders/middleware.py

```diff
@@ -62,9 +62,11 @@ def __call__(
     async def __acall__(self, request: HttpRequest) -> HttpResponseBase:
         response = self.check_preflight(request)
         if response is None:
-            result = self.get_response(request)
-            assert not isinstance(result, HttpResponseBase)
-            response = await result
```

Comment:
This section did not need changing. The change lead to insufficient test coverage.

It doesn't need changing because when `__acall__` is called, we know we're in async mode, and that `get_response` will return a coroutine. Mypy just doesn't know that, hence the [type-narrowing `assert`](https://adamj.eu/tech/2021/05/17/python-type-hints-how-to-narrow-types-with-isinstance-assert-literal/)